examples/README.md

```diff
@@ -0,0 +1,43 @@
+# Examples
+
+This directory contains example notebooks and scripts demonstrating how to use PyBOP.
+
+## Directory Structure
+
+- `notebooks/`: Jupyter notebooks of example functionality with explanations
+- `scripts/`: Python scripts for quick reference and command-line usage
+- `standalone/`: Example scripts for using standalone classes
+
+## Notebooks
+
+The `notebooks/` directory contains Jupyter notebooks that provide detailed, interactive examples of various features and use cases. These notebooks include explanations, code snippets, and visualisations.
+
+To view the notebooks with interactive figures without downloading the repository, please use nbviewer:
+
+<div align="center">
+
+[Notebooks on nbviewer](https://nbviewer.org/github/pybop-team/PyBOP/tree/develop/examples/notebooks/)
```

Comment:
Should this direct to `main`? Please also add a link in the top level Readme.

Also, it looks like the links would remain static, so perhaps we could include links at the top of each notebook so it is possible to view any specific notebook in one click from GitHub?